### PR TITLE
Feature/19357 seperation of python agents

### PR DESCRIPTION
**Pull request by istherepie**
Python agents seperation from OIO rest package (except saml2.py dependency).
Legacy agents excluded from installation.
Documentation (status & todo) added.

### CODE REVIEW

**Comment by istherepie** [comment]
Jeg har tilføjet ændringer jf. Carsten's kommentarer.

File: python_agents/README.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MOX python agents
+=================
+
+This directory contains the following 2 (legacy) mox agents:
+    * MOX Advis (mox_advis.py)
```
Comment (danchr) [inline python_agents/README.rst]: Stylistically, I'd suggest using monospaced text for file names either an em-dash (—) for separation or perhaps a table.

File: python_agents/README.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    * MOX Elk Log (mox_elk_log.py)
+
+These agents were part of the early iterations of the Lora stack.
+However these agents not used in the current release (may be added in the forseeable future).
+
+As such the agents have been excluded from the default installation procedure
+until the code base is evaluated and updated.
```
Comment (danchr) [inline python_agents/README.rst]: I'd suggest compressing this a bit:

“The two agents are currently unsupported and not part of the default installation procedure.”

File: python_agents/README.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+Update/Dependencies
+-------------------
+
+MOX Advis & MOX Elk Log have previouly been coupled with the OIO Rest package.
+The agents have now been de-coupled and do not depend on external settings and python dependencies.
+
+However the agents depend on a local OIO Rest module: saml2.py (Saml2_Assertion class).
+This dependency is not Python 3 compatible
+and will be removed or refactored when the code is moved to Python 3.
```
Comment (danchr) [inline python_agents/README.rst]: 👍

File: install.sh [hunk 4]
```diff
@@ -88,7 +88,10 @@ echo "$DIR/agentbase/python/mox" > "$DIR/agentbase/python/mox/mox.pth"
 # Compile agents
 echo "Installing Agents"
 $DIR/agents/install.sh "$DOMAIN"
-$DIR/python_agents/install.py
+
```
[the remark this answers is not in the document]
Comment (danchr) [reply install.sh]: Agreed.

**Comment by istherepie** [comment]
Suggested changes applied.

**Pull request merged**